scripts/eurostat/gdp/eurostat_gdp.mcf

```diff
@@ -1,14 +1,17 @@
-Node: dcid:Amount_EconomicActivity_GrossDomesticProduction
+Node: dcid:Amount_EconomicActivity_GrossDomesticProduction_Nominal
 typeOf: dcs:StatisticalVariable
 populationType: dcs:EconomicActivity
 activitySource: dcs:GrossDomesticProduction
 measuredProperty: dcs:amount
 statType: dcs:measuredValue
+measurementQualifier: dcs:Nominal
 
-Node: dcid:Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person
+
+Node: dcid:Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person_Nominal
```

Comment:
@fpernice-google for the time being, can you change this node's ID to match [Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita](https://browser.datacommons.org/browser/Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita)

We are working on the ability to use StatVars that are already checked into prod (so that you don't need to redefine existing ones, reducing this issue of same StatVars with diff IDs)

Reply:
This was already reimported with the `_Nominal` suffix, as discussed via chat. Should I still go back to original format? (Same for comments below.)

Reply:
Nope! Thanks.

Reply:
Done.